tests/workflow/test_workflow_dependencies.py

```diff
@@ -0,0 +1,122 @@
+# -*- coding: utf-8 -*-
+
+import pytest
+import networkx as nx
+import json
+import matplotlib.pyplot as plt
+
+from juicer.workflow.workflow import Workflow
+
+def debug_instance(instance_wf):
+    print
+    print '*' * 20
+    print instance_wf.workflow_graph.nodes()
+    print '*' * 21
+    print instance_wf.workflow_graph.edges()
+    print '*' * 22
+    print instance_wf.workflow_graph.is_multigraph()
+    print '*' * 23
+    print instance_wf.workflow_graph.number_of_edges()
+    print '*' * 24
+    print instance_wf.sorted_tasks
+    print '*' * 25
+    test = instance_wf.get_reversed_graph()
+    print test.edges()
+    print '*' * 26
+    print instance_wf.workflow_graph.in_degree()
+    print instance_wf.check_in_degree_edges()
+    print '*' * 27
+    print instance_wf.workflow_graph.out_degree()
+    print instance_wf.check_out_degree_edges()
+    print '*' * 28
+
+    # Show image
+    # pos = nx.spring_layout(instance_wf.workflow_graph)
+    # pos = nx.fruchterman_reingold_layout(instance_wf.workflow_graph)
+    # nx.draw(instance_wf.workflow_graph, pos, node_color='#004a7b', node_size=2000,
+    #         edge_color='#555555', width=1.5, edge_cmap=None,
+    #         with_labels=True, style='dashed',
+    #         label_pos=50.3, alpha=1, arrows=True, node_shape='s',
+    #         font_size=8,
+    #         font_color='#FFFFFF')
+    # plt.show()
+    # plt.savefig(filename, dpi=300, orientation='landscape', format=None,
+                 # bbox_inches=None, pad_inches=0.1)
+
+
+def test_workflow_sequence_missing_outdegree_edge_failure():
+    workflow_test = json.load(
+        open("./tests/workflow/workflow_missing_out_degree_edge.txt"),
```

Comment:
Files such this should be in a folder called fixtures. Change the name of folder to ./tests/fixtures. It's the convention we are using.